engines/query-sparql-rdfjs-lite/config/config-default.json

```diff
@@ -3,6 +3,6 @@
     "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/config-query-sparql/^4.0.0/components/context.jsonld"
```

Comment:
We forgot to also change `query-sparql-rdfjs`.

Reply:
what do you mean? it is changed there, right?

https://github.com/comunica/comunica/pull/1550/files/f4b80039964e321f9078d61f560ef38e92867d04#diff-9fc8647b28af4ac5396ff275b1ef5920c742a739004bb3970b0a24cadb5ae9daR6

Reply:
Huh, I was looking for it before, but it's there indeed, nvm then :-)